Frameworks/CoreGraphics/CGColor.mm

```diff
@@ -24,22 +24,22 @@
 const CFStringRef kCGColorWhite = CFSTR("WHITE");
 const CFStringRef kCGColorClear = CFSTR("CLEAR");
 
-static IWLazyClassLookup _LazyUIColor("UIColor");
+static IWLazyClassLookup __LazyUIColor("UIColor");
 
 /**
  @Status Caveat
  @Notes Limited constants supported
 */
-CGColorRef CGColorGetConstantColor(CFStringRef name) {
+CGColorRef _CGColorGetConstantColor(CFStringRef name) {
```

Comment:
Well I guess I should move it from the "; private exports below" section of the def then. :P